pysoa/client/settings.py

```diff
@@ -1,38 +1,8 @@
 from __future__ import unicode_literals
 
-from pysoa.common.settings import (
-    SOASettings,
-    BasicClassSchema,
-)
-from pysoa.common.transport.asgi.settings import ASGITransportSchema
-
-from conformity import fields
+from pysoa.common.settings import SOASettings
 
 
 class ClientSettings(SOASettings):
     """Settings specifically for clients."""
-
-    schema = {
-        'client': BasicClassSchema(),
-        'cacheable': fields.Boolean(),
-    }
-    defaults = {
-        'client': {'path': 'pysoa.client:Client'},
-        'cacheable': False,
-    }
-
-    def convert_client(self, value):
-        return self.standard_convert_path(value)
-
-
-class ASGIClientSettings(ClientSettings):
```

Comment:
So why isn't this schema needed anymore? Has ASGI configuration been moved somewhere else that's not in this diff?